cake-autorate.sh

```diff
@@ -97,6 +97,8 @@ cleanup_and_killall()
 	fi
 	terminate "${proc_pids[@]}"
 
+	sleep_s 1
```

Comment:
Why are you sleeping for 1sec here?

Reply:
Good point. I forgot to remove this. I've removed it now.